tests/test_browser.py

```diff
@@ -1217,7 +1217,8 @@ def test_preinitialized_webgl_context(self):
     self.btest('preinitialized_webgl_context.cpp', '5', args=['-s', 'GL_PREINITIALIZED_CONTEXT=1', '--shell-file', path_from_root('tests/preinitialized_webgl_context.html')])
 
   def test_emscripten_get_now(self):
```

Comment:
We now require that tests that use pthreads should be marked `@requires_threads` (split it into separate tests if necessary).

Also in the later tests below.